install.sh

```diff
@@ -3679,24 +3672,22 @@ find_git_hook_templates() {
 ############################################################
 mark_directory_as_target() {
     TARGET="$1"
-    if [ "$TARGET" = "" ]; then
+    if [ -z "$TARGET" ]; then
         return
     fi
 
-    if [ "$2" != "" ]; then
-        TARGET="${TARGET}/$2"
```

Comment:
Already settings this here, make the next if, not the thing the function actually wants to test! -> I rewrote, for clearer appearance